overview.md

```diff
@@ -0,0 +1,158 @@
+# SourceCred
+
+SourceCred is anchored by a simple goal: we want open-source contributors to
+earn a living from their work. At first, we focused on the obvious problem:
+where will the money come from? We were inspired by the ability of
+cryptocurrencies to fund open-source projects, and started [SourceGrain], with
+the goal of letting every open-source project issue its own token.
+
+However, as we evolved this idea, we ran into a big problem: we don't know who
+deserves credit in an open-source project, and thus we don't know how to
+distribute the rewards. The obvious approaches, like counting commits or lines
+of code, would be far too easy to game—and would in any case miss out on vital
+work like design, or issue triaging.
+
+This affects every method for funding open-source. Whether the funds come from
```

Comment:
This inability to quantify contributions accurately affects every method for funding open-source. Projects need a fair system for deciding how to distribute funds regardless of whether they come through sponsorships, donations, [cryptocurrency airdrops](https://steemit.com/free/@pokernomad/what-is-a-cryptocurrency-airdrop), or through any [other means]. SourceCred is designed to do just that.

- Unpacks what "this" is
- Put the action of the sentence before the qualifiers
- Starts with the familiar/common funding models rather than the novel, lesser known, controversial one [Choose your own link for a good definition/explanation of airdrops.]